Travis/js/main.js

```diff
@@ -0,0 +1,219 @@
+
+let wdi10_2 = [
+	{name:'kait', location: 'St.Louis'},
+	{name:'charls', location: 'south side'},
+	{name:'michael', location: 'north side'}
+]
+
+// A list of student names from our class, each with a location and each with a list of favorite tv shows. array of Objects
+ 
+ let wdi10_3 = [
+	{name:'kait', location: 'St.Louis', tvShows: 'orange is the new black'},
+	{name:'charls', location: 'south side', tvShows: 'attack on titan'},
+	{name:'michael', location: 'north side', tvShows: 'breaking bad'}
+]
+
+// Make an array that holds all of the colors of the rainbow.
+// Write code that will access "blue" from the rainbow array.
+
+
+let rainbow = ["blue","red","green","yellow"]
+console.log(rainbow[0]);
+
+
+
+// Make an object that is called your name and holds the information about your favorite food, a hobby, the name of the town that you live in currently, and your favorite datatype. Write code that will access your hobby from the object that you just created.
+
+let yourName = {
+	favFood: 'mayo',// what?!
+	hobby: 'beekeeping', 
+	town: 'hermosa town',
+	datatype: 'social'
+}
+
+console.log(yourName.hobby)
+
+
+const crazyObject = {
+  taco: [{meat: 'steak',
+         cheese: ['panela', 'queso', 'chihuahua']},
+         {meat: 'chicken',
+          salsa: ["pico", "hot", "hotter", "really hot", "really really hot", "omg my mouth is burning"]},
+        ],
+  larry: {
+    nicknames: ["LD", "Chicken Teriyaki Boyyyyyy"],
+    quotes: ["Pretty pretty prettayyyyy good", "Is that a parkinson's thing?", "women love a self confident bald man", "I'm a walking talking enigma"],
+    characters: [{
+      name: "Jeff",
+      occupation: "manager"
+    },
+    {
+      name: "funkhauser",
+      occupation: "tv dude"
+    },
+    {
+      name: "susie",
+      occupation: "jeffs wife",
+      favourtieHobby: "Swearing at Larry and Jeff"
+    },
+    ]
+  }
+}
+
+
+// Use crazy Object to log the following.
+// "omg my mouth is burning"
+console.log(crazyObject.taco[1].salsa[5]);
+// "Pretty pretty prettayyyyy good"
+console.log(crazyObject.larry.quotes[0]);
+// "Swearing at Larry and Jeff"
+console.log(crazyObject.larry.characters[2].favourtieHobby);
+// "Chicken Teriyaki Boyyyyyy"
+console.log(crazyObject.larry.nicknames[1]);
+// The object the contains the name funkhauser
+console.log(crazyObject.larry.characters[1]);
+
+
+
+
+
+// Change the value of limbo to null.
+
+
+const inception = {
+   reality: {
+       dreamLayer1: {
+           dreamLayer2: {
+               dreamLayer3: {
+                   dreamLayer4: {
+                       dreamLayer5: {
+                           dreamLayer6: {
+                               limbo: "Joseph Gordon Levitt"
+                          }
+                       }
+                   }
+               }
+           }
+       }
+   }
+}
+
+
+console.log(inception.reality.dreamLayer1.dreamLayer2.dreamLayer3.dreamLayer4.dreamLayer5.dreamLayer6.limbo = "null")
```

Comment:
oh nooooo
heartbeaker... you did all that hard work to get to the right place but then set it equal to the string "null" instead of just `null` which is it's own thing
-0.5
**0.5/1**